include/vm_phys.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _VM_PHYS_H_
+#define	_VM_PHYS_H_
+
+#include <queue.h>
+#include <common.h>
+
+#define VM_NFREEORDER 9
+#define VM_RESERVED 1
+
+typedef uintptr_t vm_addr_t;
+typedef uintptr_t vm_paddr_t;
+typedef struct vm_page* vm_page_t;
```

Comment:
As `vm_page` is at the very core of kernel they could make a mistake early enough so that changing it now would involve modification of too many lines of code. However, if you find a sensible reason behind it I'll reconsider.